<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeDialog</class>
 <widget class="QDialog" name="RecipeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Рецепт</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Описание:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="descriptionEdit">
     <property name="maximumHeight">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Время (мин):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="cookingTimeSpinBox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Сложность:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="difficultyComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Категория:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="categoryComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Ингредиенты</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="addIngredientButton">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeIngredientButton">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="ingredientsTable">
         <property name="minimumHeight">
          <number>150</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Шаги приготовления</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="addStepButton">
           <property name="text">
            <string>Добавить шаг</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeStepButton">
           <property name="text">
            <string>Удалить шаг</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="stepsTable">
         <property name="minimumHeight">
          <number>200</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Теги</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Доступные теги:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="availableTagsList"/>
         </item>
         <item>
          <widget class="QPushButton" name="addTagButton">
           <property name="text">
            <string>&gt;&gt; Добавить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Теги рецепта:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="recipeTagsList"/>
         </item>
         <item>
          <widget class="QPushButton" name="removeTagButton">
           <property name="text">
            <string>&lt;&lt; Удалить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="newTagButton">
       <property name="text">
        <string>+ Новый тег</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBox">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>RecipeDialog</receiver>
   <slot>reject()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked()</signal>
   <receiver>RecipeDialog</receiver>
   <slot>accept()</slot>
  </connection>
 </connections>
</ui>